hexkit/providers/akafka/provider.py

```diff
@@ -149,6 +199,8 @@ async def construct(
 
         producer = kafka_producer_cls(
             bootstrap_servers=",".join(config.kafka_servers),
+            security_protocol=config.security_protocol,
+            ssl_context=generate_ssl_context(config),
```

Comment:
These two args must be also added to the init method of the KafkaProducerCompatible Protocol above. Same for Consumer.